typegen.json

```
@@ -1,6 +1,6 @@
 {
   "outDir": "src/types",
-  "specVersions": "http://localhost:8888/graphql",
+  "specVersions": "https://joystream.archive.subsquid.io/graphql",
```

I think for development purposes it's better to leave it set to localhost:
```suggestion
  "specVersions": "http://localhost:8888/graphql",
```

The files generated by `make typegen` are currently comitted anyway, so they don't need to be re-generated every time.